backend/src/models/open_content.go

```diff
@@ -34,6 +34,7 @@ type OpenContentActivity struct {
 }
 
 type OpenContentFavorite struct {
+	DatabaseFields
```

Comment:
We can't have `DatabaseFields` here because in the migration, this uses the OCP id, content_id, and user_id as the primary key, and the `ID` field in DatabaseFields is annotated with `gorm:"primaryKey"`. so if it needs an ID field, you can just add one inline.

Reply:
Made the correction, I noticed that created_at was being referenced but the values were null for this column so I added DatabaseFields. The ID isn't used so I will just add the created_at and then i can remove the Unscoped() call to since we are not using the deleted_at either.

Reply:
I take it back deleted_at is being referenced. Added it back.

Reply:
do we need `deleted_at` if we are going to unscoped delete it anyway?

Reply:
Had to add it back since it is referenced within a query, 'go test -v' (creates the table but no deleted_at column is created--had to add it back to the struct for test to pass).  Not sure if deleted_at was intended to be referenced within the query? Can I just remove this from the where clause?

Reply:
yeah definitely. i probably wouldn't have added it to the table at all if i would have thought about it

Reply:
Cool, I removed all the references to deleted_at and also created the sql script for dropping deleted_at and updated_at since these are not being used/populated.  Added the migration flag to the PR.